### PR TITLE
WTD AU Landing Page

### PR DESCRIPTION
**Pull request by swapnilogale**
Setting up WTD AU 2019  site

### CODE REVIEW

**Comment by swapnilogale** [comment]
@plaindocs Can you give it a quick eye-over once all checks are done and merge it please?

**Comment by swapnilogale** [comment]
Can I merge this in @plaindocs?

**Comment by ericholscher** [comment]
Do we have a venue contract signed? I haven't seen anything, and we shouldn't announce dates until that happens.

File: docs/conf/australia/2019/team.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+{% include "bios/mikey.rst" %}
+
+Hillary Frayley
+~~~~~~~~~~~~~~~
+
+{% include "bios/hillary.rst" %}
+
+Bio coming soon
+
```
Comment (ericholscher) [inline docs/conf/australia/2019/team.rst:64]: This page needs to be updated.

File: docs/conf/australia/2019/team.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Primary Roles
+-------------
+
+* **Conference chair** - Eric Holscher
+* **Conference co-chair** - *TBD*
+* **Code of conduct response** - See `code of conduct <http://www.writethedocs.org/code-of-conduct/#staff-contact-information>`_ page
```
Comment (ericholscher) [inline docs/conf/australia/2019/team.rst]: This URL needs to be updated. 

```suggestion
* **Code of conduct response** - See `code of conduct <http://www.writethedocs.org/code-of-conduct/#australia-code-of-conduct-team>`_ page
```

File: docs/conf/australia/2019/welcome-wagon.rst [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+----
+
+.. contents::
+   :local:
+   :backlinks: none
+
+Where is everything?
+~~~~~~~~~~~~~~~~~~~~
+
+The conference main stage is in the Crystal Ballroom on the third floor at 1332 West Burnside Street.
```
Comment (ericholscher) [inline docs/conf/australia/2019/welcome-wagon.rst:61]: This isn't correct.

**Review by ericholscher — COMMENTED** [review]
We probably need to go through and delete a bunch of Portland references and update them. Should probably do a search & replace for portland.

**Comment by swapnilogale** [comment]
@ericholscher I will update all of these over the next few weeks. Was just planning to do the landing page at this stage (minus all of these pages). Is that possible?

**Comment by ericholscher** [comment]
Ah yea, that should be fine. Just need to make the CI pass then (fix the COC link) 👍

**Comment by ericholscher** [comment]
I'll give this a peek tomorrow and merge if it looks good 👍

**Comment by swapnilogale** [comment]
Thanks Eric.

**Comment by plaindocs** [comment]
Getting some warnings from Netlify about content served over http not https. No big deal, but we should prob find and fix at some point.

**Pull request merged**